ansible/templates/sir_complainsalot-params-prod.json.j2

```diff
@@ -12,11 +12,6 @@
       "maintainers": ["U01AP8GSWG3","U01069KCRS7"],
       "retired-signals": ["smoothed_covid19","smoothed_adj_covid19"]
     },
-    "chng": {
-      "max_age": 6,
-      "maintainers": ["U01AP8GSWG3","U01069KCRS7"],
-      "retired-signals": ["7dav_outpatient_covid","7dav_inpatient_covid"]
-    },
```

Comment:
suggestion (optional): Maybe just comment out the CHNG stuff for the time being instead of removing it entirely, so it's easy to find/reference later

Reply:
So til you cannot comment in json files.